src/fluree/db/json_ld/api.cljc

```diff
@@ -359,13 +359,13 @@
   ([db q opts]
    (promise-wrap (query-api/query db q opts))))
 
-(defn from-query
+(defn query-connection
```

Comment:
This is nitpicky but I feel like this name change makes this fn's purpose less obvious. I originally chose `from-query` because that was the fundamental difference from kind of query the `query` fn took. I.e. it takes one or more ledgers in a `from` key. And since the query itself embeds the ledger(s) to use, you have to call it on a conn.

But is there a reason I'm missing this works better (now)?

Reply:
I chose this name because this action queries a connection object as opposed to a db object. With that scheme, the generic `query` function could be called `query-db`, and we could theoretically also have a `query-ledger` that loads the latest known db for that ledger and just calls `query`/`query-db`. 

I didn't think `from-query` was adequate because this patch added functionality. This function doesn't just handle `from` queries anymore; it also handles `from-named` queries too. Those are fundamentally different concepts but also must interact within the same query, and the interaction is more subtle than can be accomplished by two separate functions. Leaving the name as `from-query` would obfuscate that this function does this extra thing.